openlibrary/templates/type/edition/view.html

```diff
@@ -150,7 +150,8 @@
       </div>
 
       <div class="work-line">
-        $_("An edition of") <a href="$work.key?edition=$(ocaid)">$work_title</a>
+        $_("An edition of") <a href="$work.key?edition=$(ocaid)">$work_title$cond(work.subtitle, ': %s' % work.subtitle)
+        </a>
```

Comment:
Sight still weird whitespace!

![image](https://user-images.githubusercontent.com/6251786/113450171-59da7580-93cd-11eb-8356-3a6fb125548c.png)


```suggestion
        $_("An edition of") <a href="$work.key?edition=$(ocaid)">$work_title$cond(work.subtitle, ': %s' % work.subtitle)</a>
```